inventory/sample/group_vars/all/openstack.yml

```diff
@@ -35,6 +35,13 @@
 #   - ""
 # external_openstack_metadata_search_order: "configDrive,metadataService"
 
+## Application credentials to authenticate against Keystore API
```

Comment:
What is Keystore? Do you mean Keystone?

Reply:
Good catch !Thanks @rptaylor !
I'll fix that asap